apps/fabric-website/homepage.htm

```diff
@@ -1,7 +1,7 @@
 <div
   id="metadata"
   title="Home - Fluent UI"
-  description="The official front-end framework for building experiences that fit seamlessly into Office and Office 365."
+  description="The official front-end framework for building experiences that fit seamlessly into Microsoft 365."
```

Comment:
We probably need to do a larger pass throughout the repo to fix lotttsssss more instances, but this is a good start. 👍

Reply:
yeah...website copy is clean of incorrect 'office' stuff